src/messages.rs

```diff
@@ -347,8 +347,8 @@ pub enum MessageContent {
     GetNodeNameResponse {
         /// Supplied `PublicId`, but with the new name
         relocated_id: PublicId,
-        /// Our close group `PublicId`s.
-        close_group_ids: Vec<PublicId>,
+        /// The routing table structure of our group, including the `PublicId`s of our contacts.
+        groups: Vec<(Prefix<XorName>, Vec<PublicId>)>,
```

Comment:
is `groups` is actually the whole routing_table structure of this node, not just `its group` ?